catalog/core/catalog-core-solr/pom.xml

```diff
@@ -29,6 +29,11 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-simple</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Just curious, why was this moved up?

Reply:
Usually this is done to have a dependency take priority over another dependency's transitive dependencies.